scripts/release/helpers/requirements.js

```diff
@@ -0,0 +1,84 @@
+'use strict'
+
+/* eslint-disable max-len */
+
+const { capture, fatal } = require('./terminal')
+
+// Check that the `git` CLI is installed.
+function checkGit () {
+  try {
+    capture('git --version')
+  } catch (e) {
+    fatal(
+      'The "git" CLI could not be found.',
+      'Please visit https://git-scm.com/downloads for instructions to install.'
+    )
+  }
+}
+
+// Check that the `branch-diff` CLI is installed.
+function checkBranchDiff () {
+  try {
+    capture('branch-diff --version')
+  } catch (e) {
+    const link = [
+      'https://datadoghq.atlassian.net/wiki/spaces/DL/pages/3125511269/Node.js+Tracer+Release+Process',
+      '#Install-and-Configure-branch-diff-to-automate-some-operations'
+    ].join('')
+    fatal(
+      'The "branch-diff" CLI could not be found.',
+      `Please visit ${link} for instructions to install.`
+    )
+  }
+}
+
+// Check that the `gh` CLI is installed and authenticated.
+function checkGitHub () {
+  if (!process.env.GITHUB_TOKEN) {
```

Comment:
`gh` CLI [could use both](https://cli.github.com/manual/gh_help_environment) `GH_TOKEN` and `GITHUB_TOKEN` (in order of preference). It should check `GH_TOKEN` is not present in env vars to fail.

Reply:
Fixed.